source/math/FloatPoint2D.ooc

```diff
@@ -40,8 +40,8 @@ FloatPoint2D: cover {
 	round: func -> This { This new(this x round(), this y round()) }
 	ceiling: func -> This { This new(this x ceil(), this y ceil()) }
 	floor: func -> This { This new(this x floor(), this y floor()) }
-	minimum: func (ceiling: This) -> This { This new(this x minimum(ceiling x), this y minimum(ceiling y)) }
-	maximum: func (floor: This) -> This { This new(this x maximum(floor x), this y maximum(floor y)) }
+	minimum: func (ceiling: This) -> This { This new(Float minimum(this x, ceiling x), Float minimum(this y, ceiling y)) }
+	maximum: func (floor: This) -> This { This new(Float maximum(this x, floor x), Float maximum(this y, floor y)) }
```

Comment:
Before the changes the methods returned the second parameter, calling minimum(a,b) would return b, regardless of the x- and y-magnitudes of the parameters. With the changes the methods return a FloatPoint2D consisting of the largest/smallest magnitude of x and y .

Example:
Given the points a=(-1,1) and b=(1,-1), calling minimum(a,b) would give the following result:
Before changes: (1,-1)
After changes: (-1,-1)